modal/_function_utils.py

```diff
@@ -144,18 +144,19 @@ def __init__(self, f, serialized=False, name_override: Optional[str] = None):
     def get_mounts(self) -> Dict[str, _Mount]:
         if self.type == FunctionInfoType.PACKAGE:
             mounts = {
-                self.base_dir: _Mount(
-                    local_dir=self.base_dir,
-                    remote_dir=self.remote_dir,
+                self.base_dir: _Mount.local_dir(
+                    self.base_dir,
+                    remote_path=self.remote_dir,
                     recursive=True,
                     condition=package_mount_condition,
                 )
             }
         elif self.type == FunctionInfoType.FILE:
+            remote_path = PurePosixPath(ROOT_DIR) / Path(self.file).name
```

Comment:
nit: ROOT_DIR is already a PurePosixPath, so no need to wrap it in another one

Reply:
Thanks, will fix!